trivia-game/client/src/components/header.js

```diff
@@ -0,0 +1,20 @@
+const Header = (props) => {
+
+  let message1;
+  let message2;
+  if (props.user) {
+    message1 = `WELCOME TO TANYA'S GAMEBOI`;
+    message2=  `Game on, ${props.user}!`;
+  } else {
+    message1 = `WELCOME TO TANYA'S GAMEBOI`;
+  }
+  
+  return (
+    <div className={"textHeader"}>
+      <h2>{message1}</h2>
```

Comment:
In fact, because it does not change, you don't really even need the `message1` variable at all. This could instead be `<h2>WELCOME TO TANYA'S GAMEBOI</h2>`.